pkgs/by-name/mu/multiqc/package.nix

```diff
@@ -30,6 +30,15 @@ python3Packages.buildPythonApplication rec {
     })
   ];
 
+  # Multiqc cannot remove temporary directories in some case.
+  # Default is 10 retries, lower it to 2
+  postPatch = ''
+    substituteInPlace multiqc/utils/util_functions.py \
+      --replace-fail \
+        max_retries: int = 10, \
+        max_retries: int = 2,
```

Comment:
```suggestion
        "max_retries: int = 10," \
        "max_retries: int = 2,"
```

Reply:
Sorry, forget to rebuild to check. Corrected and force-pushed...